crates/forge/bin/cmd/coverage.rs

```diff
@@ -342,9 +355,11 @@ impl CoverageArgs {
         for report_kind in self.report {
             match report_kind {
                 CoverageReportKind::Summary => SummaryReporter::default().report(&report),
-                // TODO: Sensible place to put the LCOV file
                 CoverageReportKind::Lcov => {
-                    LcovReporter::new(&mut fs::create_file(root.join("lcov.info"))?).report(&report)
+                        match self.report_file {
+                            Some(_) => return LcovReporter::new(&mut fs::create_file(root.join(self.report_file.as_ref().unwrap()))?).report(&report),
```

Comment:
instead of doing `Some(_)` and then using `self.report_file.as_ref()`, you can just use `Some(report_file)`—you'll have the `PathBuf` available instead of an `Option<_>` and you won't need to unwrap 😄

Reply:
good catch, thanks